Dockerfile

```diff
@@ -47,16 +49,16 @@ RUN pip install pipenv \
     cspell \
     jscpd \
     markdown-link-check \
+    && git clone https://github.com/pyenv/pyenv.git --depth=1 "${PYENV_ROOT}" \
+    && find "${PYENV_ROOT}" -type d -name ".git" -exec rm -rf {} + \
+    && eval "$(pyenv init -)" \
```

Comment:
Need do to this before the find to cleanup the plugins

Reply:
Not true; plugins don't come in via `init` 🤔

Reply:
I think I remember reading that the init loads the plug-in paths to $PATH.